README.md

```diff
@@ -1,31 +1,104 @@
+configuration file (`nebari-config.yaml`) is rendered, as well as to kick off the deployment action.
+
+Nebari aims to abstract all these complexities for its users. Hence, it is not necessary to know any of the above mentioned technologies to have your project successfully deployed.
+
+> TLDR: If you know GitHub and feel comfortable generating and using API keys, you should have all it takes to deploy and maintain your system without the need for a dedicated
+> DevOps team. No need to learn Kubernetes, Terraform, or Helm.
+
+## Nebari on-prem
+
+We also have a version of Nebari based on OpenHPC. It can be used to deploy onto HPC systems or on bare-metal. 
+
+> NOTE: This tool is currently under development. Curious? Check out the [Nebari HPC](https://github.com/Quansight/qhub-hpc) repository.
+
+
+<!-- ### :cloud: Cloud Providers
+
+Nebari offers out-of-the-box support for [Digital Ocean](https://www.digitalocean.com/), Amazon [AWS](https://aws.amazon.com/),
+[GCP](https://cloud.google.com/ "Google Cloud Provider"), and Microsoft [Azure](https://azure.microsoft.com/en-us/).
+
+![High-level illustration of Nebari architecture](docs/source/images/qhub-cloud_architecture.png)
+
+For more details, check out the release [blog post](https://www.quansight.com/post/announcing-qhub). --> 
 
-Run the development server:
+## :computer: Installation
+
+Nebari is supported by macOS and Linux operating systems (Windows is **NOT** currently supported). It is compatible with Python 3.7+. We also encouraged using virutual environments ([`conda`](https://docs.conda.io/en/latest/), [`pipenv`](https://github.com/pypa/pipenv) or
+  [`venv`](https://docs.python.org/3/library/venv.html))
+
+Depending on where you plan to deploy, you may need to set up credentials (e.g. for cloud deployment) Once all the necessary credentials are generated, Nebari can be installed and 
+deployed in under 20 minutes! 
+
+See our [installation guide]() for a complete guide to intallation. 
+
+
+## :question: Questions?
+
+Have a look at our [FAQ](docs/source/user_guide/faq.md) to see if your query has been answered.
+
+We separate the queries for Nebari into:
+
+- [GitHub Discussions](https://github.com/nebari-dev/nebari/discussions) is used to raise discussions about a subject, such as: "What is the recommended way to do X with Nebari?"
+
+- [Issues](https://github.com/nebari-dev/nebari/issues/new/choose) for queries, bug reporting, feature requests, documentation, etc.
+
+> We work around the clock to make Nebari better, but sometimes your query might take a while to get a reply. We apologise in advance and ask you to please, be patient :pray:.
+
+## :book: Code of Conduct
+
+To guarantee a welcoming and friendly community, we require contributors to follow our [Code of Conduct](https://github.com/Quansight/.github/blob/master/CODE_OF_CONDUCT.md).
```

Comment:
As we are making the transition to a more community-driven project - we might need to revisit the CoC